frontend/src/components/App/AppContainer.tsx

```diff
@@ -2,13 +2,20 @@ import { GlobalStyles } from '@mui/material';
 import { SnackbarProvider } from 'notistack';
 import React from 'react';
 import { BrowserRouter, HashRouter } from 'react-router-dom';
-import helpers from '../../helpers';
+import helpers, { setBackendToken } from '../../helpers';
 import Plugins from '../../plugin/Plugins';
 import ReleaseNotes from '../common/ReleaseNotes/ReleaseNotes';
 import Layout from './Layout';
 import { PreviousRouteProvider } from './RouteSwitcher';
 
 export default function AppContainer() {
+  React.useEffect(() => {
```

Comment:
I think this be placed outside the component? Then it'll request and send backend token right away when the file is loaded